repository-ydb-v2/src/main/java/tech/ydb/yoj/repository/ydb/YdbRepositoryTransaction.java

```diff
@@ -171,11 +171,13 @@ private void closeStreams() {
     }
 
     private void validate(String request, StatusCode statusCode, String response) {
+        if (!isBadSession) {
```

Comment:
Exactly how dead is v1? E.g. Can we remove it from YOJ 3.0.0 in a month or two? ;-)

Reply:
> Exactly how dead is v1? E.g. Can we remove it from YOJ 3.0.0 in a month or two? ;-)

I think so :) Only 2 projects still work on v1. I'm making ping to they team lead every week. He promised me that it will be fixed in 1 month.

Reply:
Maybe we have to rename this flag as 'isTransactionClosedByServer'? Because current name is confusing

Reply:
> Maybe we have to rename this flag as 'isTransactionClosedByServer'? Because current name is confusing

done